src/delivery_reservation_lamp_manager.cpp

```diff
@@ -24,10 +24,15 @@ DeliveryReservationLampManager::DeliveryReservationLampManager(
   const rclcpp::NodeOptions & options = rclcpp::NodeOptions())
 : Node("delivery_reservation_lamp_manager", options)
 {
-  sub_state_ = this->create_subscription<autoware_state_machine_msgs::msg::StateLock>(
+  sub_reservation_lock_state_ = this->create_subscription<autoware_state_machine_msgs::msg::StateLock>(
```

Comment:
Since the naming of reservation_lock_state and the naming of reservation_state are mixed, please unify one or the other.

Reply:
Modified to be consistent with reservation_state.